financial-templates-lib/logger/SlackTransport.js

```diff
@@ -89,14 +89,22 @@ function slackFormatter(info) {
           // If the value within the object itself is an object we dont want to spread it any further. Rather,
           // convert the object to a string and print it along side it's key value pair.
           else if (typeof info[key][subKey] === "object" && info[key][subKey] !== null) {
-            formattedResponse.blocks[
-              formattedResponse.blocks.length - 1
-            ].text.text += `    - _${subKey}_: ${JSON.stringify(info[key][subKey])}\n`;
+            formattedResponse.blocks.push({
```

Comment:
The previous implementation here would act to append to the last block in the `formattedResponse` object. This is fine in most cases except that slack blocks are limited to 3000 chars per block. As a result, if a very large `subKey` object was appended to a block the slack transport could sometimes crash if this 3000 char limit was exceeded.

This change rather adds a new block to the `formattedResponse`, like done in the rest of this transport.